### ISSUE TITLE
plunker template is not working anymore

### ISSUE THREAD

**Issue by mahmood-sajjadi** [angular/material2#6439]
#### Bug, feature request, or proposal:
BUG

#### What is the expected behavior?
Plunker template should work

#### What is the current behavior?
Plunker template for material is broken
Error: (SystemJS) XHR error (404 Not Found) loading https://rawgit.com/angular/cdk-builds/master/bundles/cdk.umd.js/overlay
	Error: XHR error (404 Not Found) loading 

#### What are the steps to reproduce?
Goto https://github.com/angular/material2
Click on Plunker Template link

or

directly open flowing link:
http://plnkr.co/edit/o077B6uEiiIgkC0S06dd?p=preview
or
https://goo.gl/DlHd6U

> Providing a Plunker (or similar) is the *best* way to get the team to see your issue.
Plunker template: https://goo.gl/DlHd6U

#### What is the use-case or motivation for changing an existing behavior?
We should be able to reproduce error in plunker

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
13 Aug Angular Material Master
Last commit info: 85a6fff fix demo app (#6437)

**Comment by mahmood-sajjadi** [comment]
It seems it is solved

thank you

**Comment by danwulff** [comment]
The plunkr doesn't have the imports in alphabetical order (THE HORROR!!!) Thus, here are the imports in alphabetical order:

```javascript
// CDK Secondary entry points
'@angular/cdk/a11y': 'npm:@angular/cdk/bundles/cdk-a11y.umd.js',
'@angular/cdk/bidi': 'npm:@angular/cdk/bundles/cdk-bidi.umd.js',
'@angular/cdk/coercion': 'npm:@angular/cdk/bundles/cdk-coercion.umd.js',
'@angular/cdk/collections': 'npm:@angular/cdk/bundles/cdk-collections.umd.js',
'@angular/cdk/keycodes': 'npm:@angular/cdk/bundles/cdk-keycodes.umd.js',
'@angular/cdk/observers': 'npm:@angular/cdk/bundles/cdk-observers.umd.js',
'@angular/cdk/overlay': 'npm:@angular/cdk/bundles/cdk-overlay.umd.js',
'@angular/cdk/platform': 'npm:@angular/cdk/bundles/cdk-platform.umd.js',
'@angular/cdk/portal': 'npm:@angular/cdk/bundles/cdk-portal.umd.js',
'@angular/cdk/rxjs': 'npm:@angular/cdk/bundles/cdk-rxjs.umd.js',
'@angular/cdk/scrolling': 'npm:@angular/cdk/bundles/cdk-scrolling.umd.js',
'@angular/cdk/table': 'npm:@angular/cdk/bundles/cdk-table.umd.js',
'@angular/cdk/testing': 'npm:@angular/cdk/bundles/cdk-testing.umd.js',
```

**Comment by ichepurnoy** [comment]
Once again, all the plunks referenced at the [main website ](https://material.angular.io/) are broken. 
Cause: links are broken, likely after the update to Material version 5.0.0. 
Some plunks (like the abovementioned  [plunk](http://plnkr.co/edit/o077B6uEiiIgkC0S06dd?p=preview) ) can be fixed by fixing the version of rxjs to 5.5.2. 
Some are broken, because not all required packages are in place. 
Compare these URLS: 
[5.5.0](https://unpkg.com/@angular/cdk@5.0.0-rc0/bundles/)
[2.0.0-beta.12](https://unpkg.com/@angular/cdk@2.0.0-beta.12/bundles/)
There, at 5.0.0 some packages , like Rxjs , are missing.

@mahmood-sajjadi , can you pls look into it, and /or reopen the issue?

**Comment by ichepurnoy** [comment]
Some plunks (like the abovementioned  [plunk](http://plnkr.co/edit/o077B6uEiiIgkC0S06dd?p=preview) ) can be fixed by fixing the version of rxjs to 5.5.2, like this: 
`'rxjs': 'https://unpkg.com/rxjs@5.5.2'`.

Here's this [plunk, but fixed ](http://plnkr.co/edit/DqSN1ubvfZjjKoOuSq2j?p=preview).  I tried copy-pasting systemjs.config.js to some other broken plunks, and those begin to work too!
